src/less/toggle-button/toggle-button.less

```diff
@@ -34,6 +34,10 @@
     }
 }
 
+.toggle-button:disabled {
```

Comment:
Lets add `[aria-disabled]` to all these in case they want to disable the button but want it still to be focused. 
https://github.com/eBay/skin/blob/master/src/less/button/button.less#L19-L20
Also add aria-disabled to storybook.